follow-ups/arrays-objects/index.html

```diff
@@ -0,0 +1,647 @@
+          <p class="fragment">Jak przechować je w kodzie?</p>
+        </section>
+
+        <section>
+          <h3 style="font-size: 3rem">W osobnych zmiennych?</h3>
+          <pre class="fragment"><code class="hljs javascript">let userFirstName = 'Ewa';
+let userLastName = 'Kowalska';
+let userAge = 27;
+let userGender = 'k';
+let userEmailAddress = 'ewa.kowalska@poczta.pl';
+let userCity = 'Gdańsk';
+// ...
+</code></pre>
+          <p class="fragment">Mało to przyjazne, prawda?</p>
+        </section>
+
+        <section>
+          <h3 style="font-size: 3rem">Tablice?</h3>
+          <pre class="fragment"><code class="hljs javascript">let user = [
+  'Ewa', 'Kowalska', 27, 'k', 'ewa.kowalska@poczta.pl', 'Gdańsk'
+];</code></pre>
+          <p class="fragment">To wygląda już lepiej, ale dobranie się do np adresu email oznacza coś takiego:</p>
+          <pre class="fragment"><code class="hljs javascript">console.log( user[4] )</code></pre>
+          <p class="fragment">Pamiętanie numerków to nie jest dobry pomysł.</p>
+        </section>
+
+        <section>
+          <h3 style="font-size: 3rem">Dlatego mamy obiekty</h3>
+          <pre class="fragment"><code class="hljs javascript">let user = {
+  firstName: 'Ewa',
+  lastName: 'Kowalska',
+  age: 27,
+  gender: 'k',
+  emailAddress: 'ewa.kowalska@poczta.pl',
+  userCity: 'Gdańsk';
+}
+</code></pre>
+          <p class="fragment">A dobieramy się do wartości w ten sposób</p>
+          <pre class="fragment"><code class="hljs javascript">console.log(user.emailAddress)</code></pre>
+        </section>
+
+        <section>
+          <h3 style="font-size: 3rem">Anatomia obiektów</h3>
+          <pre class="fragment"><code class="hljs javascript">let user = { nazwa: 'wartość1', nazwa: 'wartość2'}</pre></code>
+          <p class="fragment">Porównajmy z tablicą</p>
+          <pre class="fragment"><code class="hljs javascript">let names = ["Asia", "Basia", "Kasia"]</code></pre>
+        </section>
+
+        <section>
+          <h3 style="font-size: 3rem">Wartości wewnątrz obiektów</h3>
+          <p class="fragment">Aby dostać się do wartości używamy następującej notacji</p>
+          <pre class="fragment"><code class="hljs javascript">user.nazwa</pre></code>
+          <p class="fragment">Lub, podobnie jak w przypadku tablicy</p>
+          <pre class="fragment"><code class="hljs javascript">user['nazwa']</code></pre>
+          <p class="fragment">Ale ta notacja jest rzadziej używana, tylko w specyficznych przypadkach.</p>
+        </section>
+
+
+        <section>
+          <h3 style="font-size: 3rem">Wartości wewnątrz obiektów</h3>
+          <p>Nazywane są <b>własnościami</b> lub <b>polami</b> obiektu. W angielskiej nomenklaturze określa się je jako <b>properties</b>.</p>
+          <p class="fragment">Pamiętacie <code>length</code> z tablic? To jest również pole tablic!</p>
+          <pre class="fragment"><code class="hljs javascript">console.log( names.length );</code></pre>
```

Comment:
moooże to trochę zbyt skomplikowane i abstrakcyjne, albo nieco zbyt po łebkach, że tablica też ma pole length, zupełnie jak obiekt - wydaje mi się, że może spowodować trochę mindfuck na tym etapie szkolenia u uczestniczek, może się okazać miejscem z którego powstają jakieś błędne założenia w ich głowach